<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="backToEditButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Back to edit mode</string>
          </property>
          <property name="icon">
           <iconset resource="QCoverage.qrc">
            <normaloff>:/images/reset</normaloff>:/images/reset</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="nodesButton">
             <property name="text">
              <string>Nodes</string>
             </property>
             <attribute name="buttonGroup">
              <string>computeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="edgePairButton">
             <property name="text">
              <string>Edge-Pair</string>
             </property>
             <attribute name="buttonGroup">
              <string>computeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="edgesButton">
             <property name="text">
              <string>Edges</string>
             </property>
             <attribute name="buttonGroup">
              <string>computeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="simplePathsButton">
             <property name="text">
              <string>Simple Paths</string>
             </property>
             <attribute name="buttonGroup">
              <string>computeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="primePathsButton">
             <property name="text">
              <string>Prime Paths</string>
             </property>
             <attribute name="buttonGroup">
              <string>computeButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="requirementsTab">
           <property name="statusTip">
            <string>Test Requirements Menu</string>
           </property>
           <attribute name="title">
            <string>Requirements</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="requirementsCountLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="requirementsList">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="coverageTab">
           <attribute name="title">
            <string>Coverage</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="coverageCountLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="coverageList">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Create and manipulate your graph here</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
       </property>
       <property name="rubberBandSelectionMode">
        <enum>Qt::ContainsItemShape</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="newAction"/>
    <addaction name="separator"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="nodeMenu">
    <property name="title">
     <string>Node</string>
    </property>
    <addaction name="startNodeAction"/>
    <addaction name="endNodeAction"/>
    <addaction name="startEndNodeAction"/>
    <addaction name="normalNodeAction"/>
    <addaction name="separator"/>
    <addaction name="deleteAction"/>
   </widget>
   <widget class="QMenu" name="optionsMenu">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="antialiasingAction"/>
    <addaction name="drawGridAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="exportGraphImageAction"/>
    <addaction name="separator"/>
    <addaction name="validateGraphAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="nodeMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="optionsMenu"/>
   <addaction name="toolsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/help</normaloff>:/images/help</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/about</normaloff>:/images/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/about</normaloff>:/images/about</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="antialiasingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Antialiasing</string>
   </property>
   <property name="statusTip">
    <string>Toggle Antialiasing</string>
   </property>
  </action>
  <action name="drawGridAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Draw grid</string>
   </property>
  </action>
  <action name="exportGraphImageAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/image</normaloff>:/images/image</iconset>
   </property>
   <property name="text">
    <string>Export graph to image</string>
   </property>
  </action>
  <action name="saveVisibleImageAction">
   <property name="text">
    <string>Export visible to image</string>
   </property>
  </action>
  <action name="newAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/new</normaloff>:/images/new</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create New Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/open</normaloff>:/images/open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/save</normaloff>:/images/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/save</normaloff>:/images/save</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/exit</normaloff>:/images/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="startNodeAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/start</normaloff>:/images/start</iconset>
   </property>
   <property name="text">
    <string>Starting</string>
   </property>
   <property name="statusTip">
    <string>Set node(s) to starting type</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="endNodeAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/end</normaloff>:/images/end</iconset>
   </property>
   <property name="text">
    <string>Ending</string>
   </property>
   <property name="statusTip">
    <string>Set node(s) to ending type</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="normalNodeAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/reset</normaloff>:/images/reset</iconset>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="statusTip">
    <string>Set node(s) to normal type</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/delete</normaloff>:/images/delete</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="startEndNodeAction">
   <property name="icon">
    <iconset resource="QCoverage.qrc">
     <normaloff>:/images/startend</normaloff>:/images/startend</iconset>
   </property>
   <property name="text">
    <string>Starting/Ending</string>
   </property>
   <property name="statusTip">
    <string>Set node(s) to starting/ending type</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="showLogAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show log</string>
   </property>
  </action>
  <action name="validateGraphAction">
   <property name="text">
    <string>Validate graph</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QCoverage.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="computeButtonGroup"/>
 </buttongroups>
</ui>
